x-pack/plugins/observability/public/components/app/news_feed/index.test.tsx

```diff
@@ -56,11 +57,16 @@ const newsFeedItems = [
     },
   },
 ] as NewsItem[];
+
 describe('News', () => {
   it('renders resources with all elements', () => {
-    const { getByText, getAllByText, queryAllByTestId } = render(
-      <NewsFeed items={newsFeedItems} />
-    );
+    jest.spyOn(fetcherHook, 'useFetcher').mockReturnValue({
+      data: { items: newsFeedItems },
+      status: fetcherHook.FETCH_STATUS.SUCCESS,
+      refetch: jest.fn(),
```

Comment:
If you're not asserting anything on this `refetch` there's no reason to use `jest.fn()` you can just use `() => {}`.